api/managementpb/ia/templates.proto

```diff
@@ -132,8 +133,14 @@ message CreateTemplateRequest {
 message CreateTemplateResponse {}
 
 message UpdateTemplateRequest {
+  // Machine-readable name (ID).
+  string name = 1 [
```

Comment:
Probably it's what we discussed about changing ids in beta APIs, right?

Reply:
Yeah. Even without changing ids, it is breaking change. Because name field is new and required.